### PR TITLE
feat: add producer skeleton

### PR DESCRIPTION
**Pull request by ArneD**

### CODE REVIEW

File: paket.dependencies [hunk 1]
```diff
@@ -43,7 +43,7 @@ nuget Be.Vlaanderen.Basisregisters.ProjectionHandling.Runner 9.0.0
 nuget Be.Vlaanderen.Basisregisters.ProjectionHandling.Connector 9.0.0
 nuget Be.Vlaanderen.Basisregisters.ProjectionHandling.Connector.Testing 9.0.0
 
-nuget Be.Vlaanderen.Basisregisters.Projector 10.1.1
+nuget Be.Vlaanderen.Basisregisters.Projector 11.0.0
```
Comment (lvermeulen) [inline paket.dependencies:46]: do we still need this dependency? why?

File: src/MunicipalityRegistry.Producer/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+LABEL maintainer "Digitaal Vlaanderen <digitaal.vlaanderen@vlaanderen.be>"
+
+COPY / /app
+WORKDIR /app
+
+RUN apt-get update && \
+    apt-get install curl jq -y && \
+    chmod +x ./init.sh
+
+EXPOSE 2014/tcp
```
Comment (lvermeulen) [inline src/MunicipalityRegistry.Producer/Dockerfile:12]: another port to open! :-)

**Comment by lvermeulen** [comment]
lgtm otherwise

File: src/MunicipalityRegistry.Producer/Dockerfile (hunk 2 above)
Comment (YusufMavzer) [reply src/MunicipalityRegistry.Producer/Dockerfile:12]: I'll put in the ticket from AWS

Comment (ArneD) [reply src/MunicipalityRegistry.Producer/Dockerfile:12]: Indeed

File: paket.dependencies (hunk 1 above)
Comment (ArneD) [reply paket.dependencies:46]: I've used the projector here so we can communicate with it. To get the status, ask to start/stop, etc...

**Review by YusufMavzer — APPROVED** [review]

**Pull request merged**